src/virtualenv/activation/nushell/activate.nu

```diff
@@ -49,44 +64,53 @@ export-env {
     let venv_path = ([$virtual_env $bin] | path join)
     let new_path = ($old_path | prepend $venv_path | str collect $path_sep)
 
-    # Creating the new prompt for the session
-    let virtual_prompt = if ('__VIRTUAL_PROMPT__' == '') {
-        $'(char lparen)($virtual_env | path basename)(char rparen) '
-    } else {
-        '(__VIRTUAL_PROMPT__) '
+    let new_env = {
+        $path_name  : $new_path
+        VIRTUAL_ENV : $virtual_env
     }
 
-    # Back up the old prompt builder
-    let old_prompt_command = if (has-env 'VIRTUAL_ENV') and (has-env '_OLD_PROMPT_COMMAND') {
-        $env._OLD_PROMPT_COMMAND
+    let new_env = if (is-env-true 'VIRTUAL_ENV_DISABLE_PROMPT') {
+      $new_env
     } else {
-        if (has-env 'PROMPT_COMMAND') {
-            $env.PROMPT_COMMAND
-        } else {
-            ''
-        }
-    }
+      # Creating the new prompt for the session
+      let virtual_prompt = if ('__VIRTUAL_PROMPT__' == '') {
```

Comment:
`'__VIRTUAL_PROMPT__' == ''` seems like it has an extra pair of `'`s.

Should probably be `__VIRTUAL_PROMPT__ == ''`